### PR TITLE
tutorial edits

### PR DESCRIPTION
**Pull request by sixhobbits**
Some suggested edits and expansions to some sections.

### CODE REVIEW

File: source/tutorials/ad-hoc-developer-environments.rst [hunk 1]
```diff
@@ -60,14 +63,14 @@ You can search the package list using:
     gitMinimal           git-2.25.0
 
 
-The first column is the :term:`attribute name` and the second is :term:`package name` and its version.
+The first column is the :term:`attribute name` and the second is :term:the `package name` and its version.
```
Comment (domenkozar) [inline source/tutorials/ad-hoc-developer-environments.rst:66]:
```suggestion
The first column is the :term:`attribute name` and the second is the :term:`package name` and its version.
```

File: source/tutorials/ad-hoc-developer-environments.rst [hunk 2]
```diff
@@ -77,7 +80,7 @@ Once you have the :term:`attribute name` for packages, you can start a shell:
 
 .. code:: shell-session
 
-    $ nix-shell -p git vim nano joe
+    $ nix-shell -p gitMinimal vim nano joe
```
Comment (domenkozar) [inline source/tutorials/ad-hoc-developer-environments.rst:83]: :+1:

**Comment by domenkozar** [comment]
@sixhobbits looks really good! One tiny change and you'll need to rebase to fix CLA check.

**Comment by domenkozar** [comment]
@ghuntley we're still getting CLA check to complain about the branch name, do you have any ideas?

**Comment by domenkozar** [comment]
recheckcla

**Comment by sixhobbits** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by sixhobbits** [comment]
recheckcla

**Comment by domenkozar** [comment]
recheckcla

**Comment by domenkozar** [comment]
recheckcla

**Pull request merged**